rules/groups-in-microsoft-365/rule.md

```diff
@@ -25,19 +25,19 @@ It is important to understand the differences between on-premises groups and Mic
 
 ### On-premises groups
 
-These groups are created and managed within an organization's local network, using on-premises infrastructure such as Active Directory. These groups typically have limited collaboration features and are primarily used for email distribution and security management. If we have to manage the groups, users, or laptops on-premises that are domain-joined it is easier to create a security group in AD.  
+These groups are created and managed within an organization's local network, using on-premises infrastructure such as Active Directory. These groups typically have limited collaboration features and are primarily used for email distribution and security management. If we have to manage the groups, users, or laptops on-premises that are domain-joined it is easier to create a security group in AD.
 
-***NOTE: AD groups sync to Microsoft 365 (cloud), but Microsoft 365 groups don't sync back to AD*** 
+***NOTE: AD groups sync to Microsoft 365 (cloud), but Microsoft 365 groups don't sync back to AD***
 
 ![Figure: Choosing a group type in On-Premises Active Directory ](on-prem-group.jpg)
 
-### Microsoft 365 groups
+### Groups in Microsoft 365
```

Comment:
The headings under this should be H4s if this is H3

I know it's not a new change that caused this but I'm seeing it now only

Reply:
headings fixed